sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/EvaluationOutcome.java

```diff
@@ -24,6 +22,7 @@
 
     private final Type type;
     private final String source;
+    private final MesosResource mesosResource;
```

Comment:
No rush just a thought, feel free to skip here. I'd just keep the current short names but have them return a builder:

common case: `[EvaluationOutcome.]pass(this, "description").build()`
additional data: `[EvaluationOutcome.]pass(this, "description").setMesosResource(mesosResource).build()`